.travis-install.sh

```diff
@@ -7,12 +7,8 @@ set -e # exit as soon as an error occurs
 msg=`git log --pretty=oneline -1`
 echo "Building commit: $msg" 
 
-# setup conda recipe
-wget https://github.com/cyclus/ciclus/archive/master.zip -O ciclus.zip
-unzip -j ciclus.zip "*/cyclus/*" -d conda-recipe
-
 # build
-conda build --no-test conda-recipe
+conda build --no-test conda
```

Comment:
I suggest renaming the dir to `recipe` or `conda-recipe` since not just anything conda related goes in here.